packages/common/src/index.ts

```diff
@@ -1,16 +1,9 @@
-import { request, gql } from 'graphql-request';
-import * as utils from './utils';
-
-import { Link as ReactRouterLink } from 'react-router-dom';
+export * from 'graphql-request';
+export * from 'react-query';
+export * from 'react-query/devtools';
 
-export * from './components';
+export * from './utils';
+export * from './ui';
```

Comment:
removed react-router from being shared here, and instead added it as an explicit dependency for each package and 'hoisted it' with `lerna boostrap`.

Before I did this, I was getting an error that I couldn't use `Routes` outside of a `Router` which indicated that the `Routes` I was importing in `TransactionService` was from a different package than what the `BrowserRouter` was imported from.

I'm still figuring out what exactly went wrong here and why I seem to have to have `react-router-dom` be an explicit dependency in the lerna hoisted modules rather than importing from the common package..